backend/src/main/scala/cromwell/backend/package.scala

```diff
@@ -37,10 +41,28 @@ package object backend {
   case class BackendWorkflowDescriptor(id: WorkflowId,
                                        workflowNamespace: NamespaceWithWorkflow,
                                        inputs: Map[FullyQualifiedName, WdlValue],
-                                       workflowOptions: WorkflowOptions)
+                                       workflowOptions: WorkflowOptions) {
+    val shortId = id.toString.split("-")(0)
```

Comment:
Delete, and replace usage of `handle.jobDescriptor.descriptor.shortId` with the existing `handle.jobDescriptor.descriptor.id.shortString`